Makefile

```diff
@@ -59,6 +59,35 @@ E2E_TEST_CONFIG ?=
 
 LDFLAGS = -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$(VERSION) -X sigs.k8s.io/node-feature-discovery/source.pathPrefix=$(HOSTMOUNT_PREFIX)"
 
+## multi-arch build
+# required to enable buildx
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+# build with buildx
+PLATFORMS ?= linux/amd64,linux/arm64
```

Comment:
Hmm, bike-shedding/nitting, but what do you think about renaming this to e.g. `BUILDX_PLATFORMS`, `IMAGE_ALL_PLATFORMS` or smth(?) Because if you do e.g. `make build` your build isn't affected.

Reply:
Right, will update asap.